otter/cloud_client.py

```diff
@@ -507,6 +524,25 @@ class NovaComputeFaultError(Exception):
     """
 
 
+@attributes([])
```

Comment:
Oh nope, I'm totally wrong, sorry :(  it's testing identity, so:'

```
>>> Exception('12345') == Exception('12345')
False
```

Whereas using characteristic will compare only the values passed specified in the `@attributes` decorator, ignoring others.  So:

```
>>> @attributes([])
... class MyException(Exception):
...    pass
... 
>>> MyException('message') == MyException('message')
True
>>> MyException('message') == MyException()
True
```

:|  Guess we will need `attr` after all?  Or to specify 'message' as a required attribute?